crates/solver/src/in_flight_orders.rs

```diff
@@ -89,6 +93,19 @@ impl InFlightOrders {
                 u256_to_big_uint(&order.data.buy_amount) > order.metadata.executed_buy_amount
             }
         });
+
+        tracing::debug!(
```

Comment:
Personally, I would demote this to a `trace` log. We can enable `trace` log level for this module for now.